scripts/routes/auth.ts

```diff
@@ -1,4 +1,4 @@
-import { NativeStackRouter as StackRouter, Route } from '@smartface/router';
+import { NativeStackRouter, NativeStackRouter as StackRouter, Route } from '@smartface/router';
```

Comment:
BaseRouter should be imported right, you can use the imports from page I think

Reply:
OK.